docs/app/shared/Admonition.tsx

```
@@ -0,0 +1,48 @@
+ * @param root0.type
+ * @param root0.title
+ * @param root0.children
+ */
+const Admonition = ({type, title, children}: Props) => {
+    const displayTitle = title || titleMap[type];
+
+    return (
+        <div
+            aria-label={displayTitle}
+            className={`admonition ${type}`}
+            role={"region"}
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)